demo/package.json

```diff
@@ -2,9 +2,10 @@
   "name": "osd-select-demo",
   "version": "1.0.0",
   "description": "",
+  "source": "src/index.html",
```

Comment:
That's all that was missing.

We were just in a rush, otherwise we would have seen it instantly -
<img width="802" alt="image" src="https://user-images.githubusercontent.com/10364499/187544259-63ab35ce-e54a-4d30-98e0-951aa35720bc.png">